internal/web/config/user_actions_types.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2025 Stefan Prodan.
+// SPDX-License-Identifier: AGPL-3.0
+
+package config
+
+import (
+	"fmt"
+	"slices"
+)
+
+const (
+	// UserActionReconcile is the reconcile user action.
+	UserActionReconcile = "reconcile"
+	// UserActionSuspend is the suspend user action.
+	UserActionSuspend = "suspend"
+	// UserActionResume is the resume user action.
+	UserActionResume = "resume"
+)
+
+var (
+	// AllUserActions lists all possible user actions.
+	AllUserActions = []string{
+		UserActionReconcile,
+		UserActionSuspend,
+		UserActionResume,
+	}
+)
+
+// UserActionsSpec holds the actions configuration.
```

Comment:
Let's add here auth type with the default set to `OAuth2`. We should let admins opt-in for actions under `Anonymous` if they expose the UI internally or they use some Ingress with mTLS in front.

Reply:
Fixed, thanks!